_posts/2021-02-12-generalist-specialization-and-the-impostor-syndrome.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Generalist Specialization and The Impostor Syndrome
+author: arto
+excerpt: >
+  Impostor Syndrome is a common visitor in our profession, as we are often in charge of shaping the world of the future, while boldly going to new, unknown areas. In this article I'll touch on what it is, how to recognize it, what to do about it, and how does it relate to being a generalist.
+tags:
+ - Software development
```

Comment:
The tag is case-sensitive. Right now there is five posts tagged as "Software Development" and one as "Software development". Just something to consider.

Reply:
Thanks, that was a good catch. I'll fix that.